docs/source/llms/tracing/index.rst

```diff
@@ -1277,93 +1253,6 @@ Client API
         attributes={"attribute3": "value3", "attribute4": "value4"},
     )
 
-Q: How can I see the stack trace of a Span that captured an Exception?
```

Comment:
note: the UI now renders exceptions, so we don't need to suggest programatic retrieval